build.sbt

```diff
@@ -72,5 +72,6 @@ lazy val membership = project
       "dev.zio"     %% "zio-test"        % "1.0.0-RC17" % "test",
       "dev.zio"     %% "zio-test-sbt"    % "1.0.0-RC17" % "test"
     ),
-    testFrameworks := Seq(new TestFramework("zio.test.sbt.ZTestFramework"))
+    testFrameworks := Seq(new TestFramework("zio.test.sbt.ZTestFramework")),
+    addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1")
```

Comment:
John insisted to not use this

Reply:
Interesting. Do you happen to know why?

Reply:
Yeah I know it was the first thing I added to the project when we started :) @jdegoes said back then that this will be problematic with dotty and it's good for client code but not for libs.

Reply:
alright, I'll remove it. It's not used much here anyway

Reply:
Also on the line of being explicit rather than implicit inside lib code.